src/compliment/utils.clj

```diff
@@ -1,8 +1,10 @@
 (ns compliment.utils
   "Functions and utilities for source implementations."
-  (:import java.io.File java.nio.file.Files
-           (java.util.jar JarFile JarEntry)
-           java.util.function.Consumer))
+  (:import [clojure.lang Cons LazySeq Var]
+           [java.io File]
```

Comment:
I prefer `()` for imports, and spelling a FQCN directly if there is a single class imported from the package.